aws-iam-role-ec2-poweruser/variables.tf

```diff
@@ -25,6 +25,18 @@ variable "saml_idp_arn" {
   description = "The AWS SAML IDP arn to establish a trust relationship. Ignored if empty or not provided."
 }
 
+variable "iam_actions" {
```

Comment:
This is a bit of nit picking but I've been trying to move to a style where related variables are encoded into objects so we inherently express their "relatedness".

Maybe something like:
```
variable iam_ec2 {
  type = "object({actions: list(string), resources: list(string)})
  description = "....."
}
```